### PR TITLE
Map VAE params without AIT_AutoencoderKL

### PR DESCRIPTION
**Pull request by hlky**
Mapping VAE params without AIT_AutoencoderKL, this removes the dependency on AITemplate package for loading weights to modules compiled without included constants, which was needed to improve user experience of [this plugin/demo repo](https://github.com/hlky/AIT)

### CODE REVIEW

File: examples/05_stable_diffusion/src/compile_lib/compile_vae.py [hunk 1]
```diff
@@ -22,62 +22,103 @@
...
-            mapped_pt_params[ait_name] = pt_params[pt_name]
+def torch_dtype_from_str(dtype: str):
+    if dtype == "float16":
+        torch_dtype = torch.float16
+    elif dtype == "float32":
+        torch_dtype = torch.float32
+    else:
+        raise ValueError("dtype not supported yet!")
+    return torch_dtype
+
```
Comment (hl475) [inline examples/05_stable_diffusion/src/compile_lib/compile_vae.py:33]: Please another empty line here

File: examples/05_stable_diffusion/src/compile_lib/compile_vae_alt.py [hunk 2]
```diff
@@ -22,98 +22,103 @@
...
-def map_vae_params(ait_module, pt_module, batch_size=1, seq_len=4096):
+def torch_dtype_from_str(dtype: str):
+    if dtype == "float16":
+        torch_dtype = torch.float16
+    elif dtype == "float32":
+        torch_dtype = torch.float32
+    else:
+        raise ValueError("dtype not supported yet!")
+    return torch_dtype
+
```
Comment (hl475) [inline examples/05_stable_diffusion/src/compile_lib/compile_vae_alt.py:33]: same for adding another empty line

File: examples/05_stable_diffusion/src/compile_lib/compile_vae.py [hunk 3]
```diff
@@ -22,62 +22,103 @@
...
+def map_vae(pt_module, device="cuda", dtype="float16"):
```
Comment (hl475) [inline examples/05_stable_diffusion/src/compile_lib/compile_vae.py:34]: Is it possible that we could simplify this method a bit? (sorry I know the complex logic is probably existing before this PR, but highly appreciate if you can help simplify as you doing the change in this PR)

File: examples/05_stable_diffusion/src/compile_lib/compile_vae_alt.py [hunk 4]
```diff
@@ -22,98 +22,103 @@
...
+def map_vae(pt_module, device="cuda", dtype="float16"):
```
Comment (hl475) [inline examples/05_stable_diffusion/src/compile_lib/compile_vae_alt.py:34]: same for simplify this method

**Review by hl475 — COMMENTED** [review]
Thanks @hlky for the PR! The code looks good to me with some minor comments.

**Review by hl475 — APPROVED** [review]
LG, thanks!

**Review by houseroad — APPROVED** [review]

**Pull request closed without merging**